src/containers/VerifyActivities.jsx

```diff
@@ -145,19 +212,20 @@ class VerifyActivities extends Component {
    */
   renderLayout() {
     const {
-      activities,
+      filteredActivities,
       showUserDetails,
       isSelectAllChecked,
       selectedActivities,
       showButtons,
     } = this.state;
     const page = this.props.history.location.pathname;
```

Comment:
line 221 and 222 could be combined as 👍 
`let {history: {location: {pathname}}, userRoles} = this.props`

feel free to push back on this if it seems unreadable.

Reply:
^ this is fine destructuring right there 👍